README.md

```diff
@@ -1,2 +1,6 @@
 # docker-helm-client
 Helm client as a Docker image to run on CIs or anywhere it seems necessary
+
+```bash
+$ docker run --rm -ti cobli/helm-client:3.2.1 version
+version.BuildInfo{Version:"v3.2.1", GitCommit:"e11b7ce3b12db2941e90399e874513fbd24bcb71", GitTreeState:"clean", GoVersion:"go1.13.10"}```
```

Comment:
We should put the ` ``` ` marker in the next line
I Can't use the suggestion feature here because it uses this marker as delimiter for suggestions 8( !